packages/lib/components/card/card.ts

```diff
@@ -0,0 +1,201 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+@customElement('cx-card')
+export class Card extends LitElement {
+  static styles = css`
+    :host {
+      display: block;
+    }
+
+    .card {
+      position: relative;
+      height: 440px;
+      width: 100%;
+      max-width: 500px;
+      border-radius: 24px;
+      overflow: hidden;
+      display: flex;
+      flex-direction: column;
+      text-decoration: none;
+      color: inherit;
+    }
+
+    .card.clickable {
+      cursor: pointer;
+    }
+
+    .card-image {
+      width: 100%;
+      height: 50%;
+      position: relative;
+      flex-shrink: 0;
+    }
+
+    .card-image img,
+    .card-image ::slotted(img),
+    .card-image ::slotted(video),
+    .card-image ::slotted(*) {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+      transition: filter 0.3s ease;
+    }
+
+    /* Blue filter on img for hover */
+    .card-image::after {
+      content: "";
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      mix-blend-mode: multiply;
+      background: var(--cx-color-background-accent-5);
+      opacity: 0;
+      transition: opacity 0.3s ease;
+    }
+
+    .card-info-wrapper {
+      flex: 1;
+      padding: var(--cx-spacing-6);
+      color: var(--cx-color-text-primary);
+      background-color: var(--cx-color-background-accent-1-soft);
+    }
+
+    .card-info {
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+    }
+
+    .card-subtitle {
+      display: flex;
+      flex-wrap: wrap;
+      color: var(--cx-color-text-less-important);
+      gap: var(--cx-spacing-2);
+      margin-bottom: var(--cx-spacing-4);
+    }
+
+    .card-title {
+      display: -webkit-box;
+      -webkit-box-orient: vertical;
+      -webkit-line-clamp: 2;
+      line-clamp: 2;
+      max-height: 65px;
+      min-height: 65px;
+      overflow: hidden;
+      font-size: 24px;
```

Comment:
Vi burde unngå font size i px verdier, siden da er det ikke mulig for bruker å endre font size i browseren. Vi har klasser for alle font size og line heights for slike formål :)